<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="TargetLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>571</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>801</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>801</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Font Size</string>
      </property>
     </widget>
     <widget class="SizeSlider" name="horizontalSizeSlider">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>441</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>119</y>
       <width>801</width>
       <height>21</height>
      </rect>
     </property>
     <widget class="FontFamily" name="ComicSansMS_Chk">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>0</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Comic Sans MS</string>
      </property>
     </widget>
     <widget class="FontFamily" name="TimesNewRoman_Chk">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>0</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Times New Roman</string>
      </property>
     </widget>
     <widget class="FontFamily" name="Arial_Chk">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Arial</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Family</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>801</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Font Size Type</string>
      </property>
     </widget>
     <widget class="FontSizeCombo" name="FontSize_comboBox">
      <property name="geometry">
       <rect>
        <x>468</x>
        <y>0</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>small</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>big</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>801</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Font Size Type</string>
      </property>
     </widget>
     <widget class="TargetLineEdit" name="FontSize_lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TargetLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">targetlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>FontSizeCombo</class>
   <extends>QComboBox</extends>
   <header location="global">fontsizecombo.h</header>
  </customwidget>
  <customwidget>
   <class>FontFamily</class>
   <extends>QRadioButton</extends>
   <header location="global">fontfamily.h</header>
   <slots>
    <signal>stateChanged(int)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>SizeSlider</class>
   <extends>QSlider</extends>
   <header location="global">sizeslider.h</header>
   <slots>
    <slot>sizeSliderChanged(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
